### PR TITLE
[docs] Wiki migration - Glossary

### LINKED ISSUE #7119: [docs] Wiki Migration
Related to #7053. Migrate current GitHub Wiki to LORIS ReadtheDocs.

### PR DESCRIPTION
**Pull request by haoweiqiu**
## Brief summary of changes

- [x] Have you updated related documentation?

#### Link(s) to related issue(s)

* Resolves #7119  (Reference the issue this fixes, if any.)

### CODE REVIEW

File: readthedocs/mkdocs.yml [hunk 1]
```diff
@@ -32,12 +32,15 @@ nav:
     - Developers:
         - Overview: 'docs/wiki/99_Developers/README.md'
         - 'Coding Standards': 'docs/CodingStandards.md'
+        - 'SQL Modeling Standard': 'docs/SQLModelingStandard.md'
```
Comment (christinerogers) [inline readthedocs/mkdocs.yml:35]: nice catch 👍

File: docs/wiki/99_Developers/LORIS_Glossary.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+**Administration:** in an instrument form, can be marked “None”, “Partial”, or “All”. If an instrument was not administered, it is marked as “None.” This will remove the instrument from the database statistics calculation. If an instrument was partially administered (i.e. candidate did not answer some questions, parent did not provide some answers, tester did not ask some of the questions), it is marked as “Partial”. This field has to be set for Data Entry to be marked as “Complete”.
```
Comment (christinerogers) [inline docs/wiki/99_Developers/LORIS_Glossary.md:1]: Can you add an introductory line like : 
`Below is a list of terms used mostly in the context of the LORIS user-facing front-end used.  For definition of Study variables used in configuring LORIS also see ` [ then add  a link to the Study Variables part where project, subproject/cohort etc are defined. ]

**Review by christinerogers — CHANGES_REQUESTED** [review]
Thanks @haoweiqiu -- good start on this one.  See my comments below. 
It's possible the Reporting Issues content will seem redundant at the end of this migration process, but let's keep it for now.

**Review by christinerogers — CHANGES_REQUESTED** [review]
Hi Haowei, 
Could you do 3 things to this PR? 
1 - fix the typo below, and re-read the document again to ensure it reads smoothly
2 - put the word "Glossary" in this PR title, since you're only migrating the one file and some meta changes here.
3 - also edit the Glossary file in the deprecated_wiki directory so that we know that content has been covered (see older examples of migrating wiki content to show you how)
And (4) please mark this as Ready for review - it's currently in draft mode.  
thanks!

**Comment by laemtl** [comment]
@haoweiqiu Is this ready for review? If so, can you change the status of the PR (currently in draft mode)?

**Comment by haoweiqiu** [comment]
@laemtl Yes, it is.

**Review by christinerogers — APPROVED** [review]
Looks good to me - thanks @haoweiqiu 
approving

**Comment by christinerogers** [comment]
@driusan please feel free to review (and merge)

File: docs/wiki/99_Developers/Reporting_Issues.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Reporting Issues
+
+Our [GitHub Issues page](https://github.com/aces/Loris/issues) is our designated forum for feedback and issues specific to the codebase.
+> For general support and questions for the community, please use our LORIS Developers mailing list: loris-dev@bic.mni.mcgill.ca
+
+## Reporting Bugs
+If you find a bug, please create a [GitHub issue](https://github.com/aces/Loris/issues) that describes where in application the bug occurred, what happened, and what you expected to happen. You can also give the issue the "bug" label. If the bug is specific to a module, please start your issue title with the module name.
```
Comment (driusan) [inline docs/wiki/99_Developers/Reporting_Issues.md:7]: I don't think people who are outside of the org can give labels?

File: docs/wiki/99_Developers/LORIS_Glossary.md [hunk 4]
```diff
@@ -1,22 +1,24 @@
-> Under Construction
+# LORIS Glossary
+
+Below is a list of terms used mostly in the context of the LORIS user-facing front-end used. For definition of Study variables used in configuring LORIS also see [here](https://acesloris.readthedocs.io/en/latest/docs/wiki/01_STUDY_PARAMETERS_SETUP/01_Study_Variables/00_Introduction_to_Study_Variables.html).
```
[the remark this answers is not in the document]
Comment (driusan) [reply docs/wiki/99_Developers/LORIS_Glossary.md:3]: Having looked further below, the terms don't seem to be frontend related either. There's a lot of backend terms.

**Review by driusan — CHANGES_REQUESTED** [review]

File: docs/wiki/99_Developers/LORIS_Glossary.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (christinerogers) [reply docs/wiki/99_Developers/LORIS_Glossary.md]: updated this since Haowei has left us.

**Review by christinerogers — APPROVED** [review]

**Comment by cmadjar** [comment]
@ridz1208 could you look at this one too? Thanks!

**Review by ridz1208 — APPROVED** [review]

**Comment by christinerogers** [comment]
I think this is ready for @driusan 's re-review

File: docs/wiki/99_Developers/Reporting_Issues.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Reporting Issues
```
Comment (driusan) [inline docs/wiki/99_Developers/Reporting_Issues.md:1]: What does this have to do with a glossary?

**Review by driusan — CHANGES_REQUESTED** [review]

File: docs/wiki/99_Developers/Reporting_Issues.md (hunk 5 above)
Comment (haoweiqiu) [reply docs/wiki/99_Developers/Reporting_Issues.md:1]: Hi @driusan, this was part of the wiki migration task, but was mixed in this PR. Thank you!

Comment (driusan) [reply docs/wiki/99_Developers/Reporting_Issues.md:1]: This file is still here.

**Comment by cmadjar** [comment]
@PapillonMcGill could you take this over please?
TODO: Remove docs/wiki/99_Developers/Reporting_Issues.md and its link in the docs (mkdocs.yml).

**Comment by PapillonMcGill** [comment]
from Travis:
`There was 1 error:

1) TestInstrumentTestIntegrationTest::testSelectOptionElement
Facebook\WebDriver\Exception\ElementClickInterceptedException: Element <select name="consent"> is not clickable at point (1158,625) because another element <div class="panel-body"> obscures it

/app/vendor/php-webdriver/webdriver/lib/Exception/WebDriverException.php:93
/app/vendor/php-webdriver/webdriver/lib/Remote/HttpCommandExecutor.php:371
/app/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php:604
/app/vendor/php-webdriver/webdriver/lib/Remote/RemoteExecuteMethod.php:27
/app/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebElement.php:74
/app/vendor/php-webdriver/webdriver/lib/WebDriverSelect.php:231
/app/vendor/php-webdriver/webdriver/lib/WebDriverSelect.php:110
/app/test/test_instrument/test_instrumentTest.php:213
`

This PR as nothing to do with HTML or JavaScript. What could be wrong?

**Comment by laemtl** [comment]
@PapillonMcGill this error occurs because of a race condition. Nothing to do with this PR, I got the same issue and rerunning the tests can help passing the faulty test. I plan to investigate later.

**Comment by PapillonMcGill** [comment]
thanks @laemtl  I jhad previously restarted but failed again. Hope this one will work

**Comment by ridz1208** [comment]
@PapillonMcGill could you fix the PR ? https://github.com/aces/Loris/pull/7120/files/7e0171f65bc7f677b9e24d661c883929fb67c942 

Dave's comment was not addressed

**Comment by PapillonMcGill** [comment]
@ridz1208 I did removed text that as been flag as `not related` by @driusan.
The `docs/wiki/99_Developers/Reporting_Issues.md ` file is still there but reduced to the placeholder stub it was before.

**Comment by ridz1208** [comment]
@driusan this one is in your court. Your comment has been addressed
